twinkle.js

```diff
@@ -20,7 +20,7 @@
 
 /* global Morebits */
 
-(function (window, document, $, undefined) { // Wrap with anonymous function
```

Comment:
Does anyone know why on earth this was here? Since my changes removed the only two mentions of undefined in the file, this here was being flagged by the linter as an unused variable(!).

Reply:
See https://stackoverflow.com/a/15777832/2521092, and 2c64199f87fff2ba04970488542f199415e6b2e6 and 6d83b686496578d5fdc0beb0b2551a9d3986d8a7 for the reasoning for adding them to Twinkle.

I looked into the various anonymous wrapper functions when I was doing #500; I don't remember everything I learned, but it basically boils down to the fact that pre-ES5, `undefined` wasn't read-only.  It was good practice to define it to prevent bad people from being bad.

See also the eslint rules [no-undefined](https://eslint.org/docs/rules/no-undefined), [no-global-assign](https://eslint.org/docs/rules/no-global-assign), and [no-shadow-restricted-names](https://eslint.org/docs/rules/no-shadow-restricted-names).

I think I've got a todo somewhere to try to remove/tidy the `undefined` uses up, but it's buried beneath 87 others...

Reply:
Thanks for that. So it seems removal of `undefined` here is ok.